docs/source/hooks/introduction.md

```diff
@@ -102,15 +102,15 @@ We recommend that you group related Hook implementations under a namespace, pref
 
 #### Registering your Hook implementations with Kedro
 
-Hook implementations should be registered with Kedro using the [`<your_project>/src/<package_name>/settings.py`](../kedro_project_setup/settings.md) file under the `HOOKS` key.
+Hook implementations should be registered with Kedro using the [`<project-name>/src/<package_name>/settings.py`](../kedro_project_setup/settings.md) file under the `HOOKS` key.
 
 You can register more than one implementation for the same specification. They will be called in LIFO (last-in, first-out) order.
 
 The following example sets up a Hook so that the `after_data_catalog_created` implementation is called every time after a data catalog is created.
 
 ```python
-# <your_project>/src/<your_project>/settings.py
-from <your_project>.hooks import ProjectHooks, DataCatalogHooks
+# <project-name>/src/<project-name>/settings.py
+from <project-name>.hooks import ProjectHooks, DataCatalogHooks
```

Comment:
```suggestion
from <package_name>.hooks import ProjectHooks, DataCatalogHooks
```
in the same way that the other instances should be changed.

Reply:
Changes made in [344a6fb](https://github.com/kedro-org/kedro/pull/1753/commits/344a6fb49d232371b8f592ee5ccc35d681bee043).